website/docs/reference/widgets/menu-button.md

```diff
@@ -1,76 +1,89 @@
+| Property            | Type | Description | Code Snippet |
+| ------------------- | ---------------- |------------------- | ---------------- |
+| **Label**           | Formatting| This property sets the label of a menu item. This can be a static text value, or it can be dynamically bound to a data source. For example, you might set the label to be the name of a page or a user. | |
+| **Visible**           | Formatting/Binding | This property controls the visibility of a menu item. This can be useful for showing or hiding the menu item based on certain conditions. For example, you might show the menu item only if a user has permission to access it. For more information, see [Visible property details](/reference/widgets#visible)) | |
+| **Disabled**           | Formatting| This property allows you to turn off input to a menu item. This can be useful if you want to prevent users from interacting with the menu item in certain scenarios. For example, you might turn off a menu item when it's not relevant or applicable. For more information, see [Disabled property details](/reference/widgets#disabled).| |
+| **Icon** | Style | This property sets the icon to be used for a menu item. This can be useful for adding visual context and branding to the menu item. The icons are used from [Blueprint](https://blueprintjs.com) library. For more information, see the [list of icons available on Blueprint](https://blueprintjs.com/docs/#icons).| |
+|**Position** | Style |  This property allows you to select the position where you want to place the icon on the menu item. You can choose between left or right alignment.| |
+|**Icon color** | Style | This property sets the color of the icon for the menu item. This can be useful for matching the icon with the overall design of the menu button.| |
+|**Text color** | Style |  This property sets the color of the text for the menu item. This can be useful for making the text stand out or blending in with the background.| |
+|**Background color**| Style | This property sets the background color of the menu item. This can be useful for adding visual interest or highlighting the selected menu item.| |
 
-### Binding Properties
+The properties are described as follows:
 
-These properties allow you to bind your widget with any other widget in queries or JS objects. The following table lists all the binding properties.
+## Menu Items Source
+The "Menu Items Source" property lets you define where the menu items come from. You can either define them manually (Static) or use a dynamic source by selecting the "Dynamic" option. Dynamic option allows you to specify a data source such as a [database query](/core-concepts/data-access-and-binding/querying-a-database) or [API](/core-concepts/connecting-to-data-sources/authentication/connect-to-apis), which can be useful for providing up-to-date information in the menu.
```

Comment:
@dilippitchika - `one callout here, even for static menu items, you can still link the label or any other property to come from a query/API.` is added as part of the label, where it is specified that it could be a static text or bind to a dynamic source.
<img width="870" alt="Screenshot 2022-12-14 at 5 25 49 PM" src="https://user-images.githubusercontent.com/99252011/207589213-4b3a59f2-8f00-49c7-87b2-03a99306cac6.png">

- For the Dynamic menu, the items are fixed - only 10 right?
- The items are created as per source data - Please refer to below screenshot - Do you think I should add that each item is not created individually?
<img width="886" alt="Screenshot 2022-12-14 at 5 27 41 PM" src="https://user-images.githubusercontent.com/99252011/207589662-f08c0f97-00cc-421e-8fe7-e62098de8a0a.png">

Please confirm

Reply:
- Max 10 items, when dynamic source is selected
- Yes, the explanation in both label and dynamic is fine. 
- I was just calling it out in the menu items source section, don't know if we need to act on it

Reply:
@dilippitchika - Menu Items source - It serves as a main section - which briefs about the property and then has individual subsections to define it, so added to individual section. @danciaclara - Your thoughts?

Reply:
> * Max 10 items, when dynamic source is selected
> * Yes, the explanation in both label and dynamic is fine.
> * I was just calling it out in the menu items source section, don't know if we need to act on it

> * Max 10 items, when a dynamic source is selected - Yes, for dynamic source. Just a follow-up question -  If the source is static the user can add any number of menus, we don't have an upper bracket.

Reply:
I think it's ok to add it under Dynamic Source too. It's just a minor change in the last line. Let's modify it. Also not sure if we adding the part about max 10 items. Let's mention it if we haven't already.